eng/pipelines/libraries/outerloop-mono.yml

```diff
@@ -0,0 +1,67 @@
+trigger: none
+
+schedules:
+- cron: "0 7 * * *" # 7 AM UTC => 12 AM PST
```

Comment:
Nit - PST claims to be UTC-8:00 so I think that 7 is actually 11 PM PST.

Reply:
I see. I got confused by the fact that we're on PDT at the moment and it is `UTC-7` right? https://www.bing.com/search?q=pst+to+utc&cvid=f064666300d74b58948813db38c310e7&FORM=ANAB01&PC=U531

but I think we should leave it as 0 8 and it will be 1AM on PDT and 12AM on PST.

Reply:
Fixed.